Mathlib/FieldTheory/NormalClosure.lean

```diff
@@ -171,6 +178,16 @@ instance is_finiteDimensional [FiniteDimensional F K] :
     f.toLinearMap.finiteDimensional_range
   apply IntermediateField.finiteDimensional_iSup_of_finite
 
+/-- All `F`-`AlgHom`s from `K` to an algebraic closed field `A` factor through
+a normal closure of `K/F`. -/
+noncomputable def _root_.IsNormalClosure.algHom_equiv_algHom_isAlgClosed (A : Type*) [Field A]
+    [Algebra F L] [Algebra F A] [IsNormalClosure F K L] [IsAlgClosed A] [FiniteDimensional F K] :
+    (K →ₐ[F] L) ≃ (K →ₐ[F] A) := by
```

Comment:
I have pushed a new version using `Algebra.IsAlgebraic F K` and the splits hypothesis. In the end, I did not use the results from `SeparableDegree` to prove the equiv since I wanted it to be the natural one coming from the restriction to `L`. This implies that now I need to assume `Algebra L A`. The old version (without this assertion) can still be recovered since we can prove without this assumption that `(K →ₐ[F] L)` and `(K →ₐ[F] A)` have the same cardinality.